playpen/ansible/roles/dev/tasks/main.yml

```diff
@@ -11,24 +11,7 @@
       line: "StrictModes no"
   notify: restart sshd
 
-- name: Install Pulp dnf repository
-  get_url:
-      url: https://repos.fedorapeople.org/repos/pulp/pulp/fedora-pulp.repo
-      dest: /etc/yum.repos.d/fedora-pulp.repo
-
-- name: Enable Pulp 2.7 beta repository
-  command: dnf config-manager --set-enabled pulp-2.7-beta
-  when: pulp_27_beta_repo_enabled == false
```

Comment:
If you decide to put the repo on fedorapeople, then you can just change this to enable the nightly repo, but keep it instead of removing it.